src/parser.js

```diff
@@ -208,6 +208,10 @@ class NEXParser extends EventEmitter {
 
 				connection.checkForSecureServer = false;
 			}
+			let date;
+			const timestamp = raw.header.timestampSeconds;
+			date = new Date(timestamp * 1000);
+			packet.date = date;
```

Comment:
```suggestion
			const timestamp = raw.header.timestampSeconds;
			packet.date = new Date(timestamp * 1000);
```

Also this only works for packets coming from the `pcap-parser` library. `pcap-ng-parser` encodes timestamp data differently https://www.npmjs.com/package/pcap-ng-parser#event-data